src/game.cpp

```diff
@@ -10872,24 +10872,45 @@ void game::chat()
     }
 
     uimenu nmenu;
-    nmenu.text = std::string( _("Who do you want to talk to?") );
-
+    nmenu.text = std::string( _( "Who do you want to talk to or yell?" ) );
```

Comment:
That text is displayed whether there are NPCs or not, we could add a check to it, but it would automagically tell the player if there are NPCs around.